src/test/java/org/painandsuffer/items/armour/chest/ChainmailTest.java

```diff
@@ -0,0 +1,34 @@
+package org.painandsuffer.items.armour.chest;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class ChainmailTest {
+    @Test
+    public void whenChainmailCreatedWithDefaultDefendAndArmourDurability_thenChainmailCreatedWithDefend15AndArmourDurability100(){
+        Chainmail chainmail = new Chainmail();
+        int chainmailDefend = 15;
+        int chainmailDurability = 100;
+        assertNotNull(chainmail);
+        assertEquals(chainmail.getDefence(),chainmailDefend);
+        assertEquals(chainmail.getArmourDurability(),chainmailDurability);
+    }
+
```

Comment:
Consider adding tests for equals and hashcode to be sure that those methods are working as expected.